src/tree_schema_free.c

```diff
@@ -81,6 +81,18 @@ lysp_include_free(struct ly_ctx *ctx, struct lysp_include *include)
     FREE_ARRAY(ctx, include->exts, lysp_ext_instance_free);
 }
 
+void
+lysp_include_free_submodule(struct ly_ctx *ctx, struct lysp_include *include)
+{
+    return lysp_include_free_(ctx, include, 0);
+}
+
+void
+lysp_include_free(struct ly_ctx *ctx, struct lysp_include *include)
+{
+    return lysp_include_free_(ctx, include, 1);
+}
+
```

Comment:
Based on these code changes it seems to me these 2 functions are useless because a flag is used to determine which one is called and then they only pass a flag to `lysp_include_free_()`, so it can be called directly.